base/deprecated.jl

```diff
@@ -1155,6 +1155,10 @@ end
 @deprecate_binding UTF8String String
 @deprecate_binding ByteString String
 
+@deprecate ascii(p::Ptr{UInt8}, len::Integer) ascii(bytestring(p, len))
```

Comment:
Shouldn't this be deprecated to `let s=bytestring(p,len); !isascii(s) && error("expecting ASCII string"); s; end`?